src/components/progress-flow/progress-flow.e2e.ts

```diff
@@ -0,0 +1,7 @@
+import { ProgressFlow } from './progress-flow';
+
+describe('limel-progress-flow', () => {
+    it('builds', () => {
+        expect(new ProgressFlow()).toBeTruthy();
```

Comment:
This test does not add much value, we should probably add some proper tests :smile: (same can be said for the flow-item component)

Reply:
@TommyLindh2 said he wasn't so fluent in writing such tests, and hoped that you guys can help with this part. So don't wait for him to fix this, :) either remove the tests, or make them nicely yourselves ;)